### PR TITLE
[SDP-1001] refactor migration commands to keep a consistent naming approach, in preparation for the TSS migrations (pt 1/2)

### PR DESCRIPTION
**Pull request by marcelosalloum**
### What

The main change here was to update the migration commands so all of them have a consistent hierarchy. Here is what they look like right now:

- `db sdp migrate up | down`: for sdp per-tenant migrations
  - The old command was ~`db migrate up | down`~. Notice it had a different (global) hierarchy despite being a per-tenant command
- `db auth migrate up | down`: for auth per-tenant migrations
  - This haven't changed.
- `db admin migrate up | down`: for admin migrations that are relevant to registering and organizing the tenants
  - The old command was ~`db tenant migrate up | down`~. It was for a global thing, yet it looked less global than the first command. Also, despite being called `tenant`, it had a global scope rather than per-tenant

#### Additional changes:

- Updated the migration folders and migration table tracker to be consistent with the command names
- Refactored the code a bit for increased consistency and better organization
- Updated wording and method documentation for increased clarity. It took me a while to figure out what each command was does, so this was highly needed especially for people who will not be debugging the code.

### Why

- The commands were inconsistent
- The documentation was unclear
- In preparation for adding the tss migrations

#### PR Structure

* [x] This PR has a reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR title and description are clear enough for anyone to review it.
* [x] This PR does not mix refactoring changes with feature changes (split into two PRs otherwise).

#### Thoroughness

* [x] This PR adds tests for the new functionality or fixes.
* [x] This PR contains the link to the Jira ticket it addresses.

#### Configs and Secrets

* [x] No new CONFIG variables are required -OR- the new required ones were added to the helmchart's [`values.yaml`] file.
* [x] No new CONFIG variables are required -OR- the new required ones were added to the deployments ([`pr-preview`], [`dev`], [`demo`], `prd`).
* [x] No new SECRETS variables are required -OR- the new required ones were mentioned in the helmchart's [`values.yaml`] file.
* [x] No new SECRETS variables are required -OR- the new required ones were added to the deployments ([`pr-preview secrets`], [`dev secrets`], [`demo secrets`], `prd secrets`).

#### Release

* [x] This is not a breaking change.
* [x] **This is ready for production.**. If your PR is not ready for production, please consider opening additional complementary PRs using this one as the base. Only merge this into `develop` or `main` after it's ready for production!

#### Deployment

* [x] Does the deployment work after merging?

[`values.yaml`]: ../helmchart/sdp/values.yaml
[`pr-preview`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/stellar-disbursement-platform/backend-helm-values
[`dev`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/backend-helm-values
[`demo`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-backend-helm-values
[`pr-preview secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/externalsecrets-common-previews.yaml#L241-L346
[`dev secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/stellar-disbursement-platform-externalsecrets.yaml
[`demo secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-sdp-externalsecrets.yaml

### CODE REVIEW

File: stellar-multitenant/pkg/cli/migrate.go [hunk 1]
```diff
@@ -83,16 +83,23 @@ func MigrateCmd(databaseFlagName string) *cobra.Command {
...
+	numMigrationsRun, err := db.Migrate(databaseURL, dir, count, adminMigrations.FS, db.StellarAdminMigrationsTableName)
 	if err != nil {
 		return fmt.Errorf("running migrations: %w", err)
 	}
 
 	if numMigrationsRun == 0 {
 		log.Info("No migrations applied.")
 	} else {
-		log.Infof("Successfully applied %d migrations.", numMigrationsRun)
+		log.Infof("Successfully applied %d migrations %s.", numMigrationsRun, migrationDirectionStr(dir))
```
Comment (CaioTeixeira95) [inline stellar-multitenant/pkg/cli/migrate.go:94]: Why not only cast to string?

```suggestion
		log.Infof("Successfully applied %d migrations %s.", numMigrationsRun, string(dir))
```

File: cmd/utils/default_cmd_inner_calls.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		cmd.Parent().PersistentPreRun(cmd.Parent(), args)
+	}
+}
+
+var CallHelpCommand = func(cmd *cobra.Command, args []string) error {
+	if err := cmd.Help(); err != nil {
+		return fmt.Errorf("calling help command: %w", err)
+	}
+	return nil
+}
```
Comment (CaioTeixeira95) [inline cmd/utils/default_cmd_inner_calls.go:20]: 🚀

**Review by CaioTeixeira95 — APPROVED** [review]
LGTM

File: stellar-multitenant/pkg/cli/migrate.go (hunk 1 above)
Comment (marcelosalloum) [reply stellar-multitenant/pkg/cli/migrate.go:94]: If I use `string(dir)`, the resulting value would be either `0`or `1`:

https://github.com/rubenv/sql-migrate/blob/9f03b5b6c963a717b2b0554dd32246fc8ac342d0/migrate.go#L27-L30

**Pull request merged**